package/MDAnalysis/coordinates/PDB.py

```diff
@@ -395,22 +421,20 @@ def _read_frame(self, frame):
                 try:
                     cell_dims = np.array([line[6:15], line[15:24],
                                          line[24:33], line[33:40],
-                                         line[40:47], line[47:54]], 
-                                         dtype=np.float32)  
+                                         line[40:47], line[47:54]],
+                                         dtype=np.float32)
                 except ValueError:
                     warnings.warn("Failed to read CRYST1 record, "
                                   "possibly invalid PDB file, got:\n{}"
                                   "".format(line))
                 else:
-                    if (cell_dims == np.array([1, 1, 1, 90, 90, 90], 
-                        dtype=np.float32)).all():
```

Comment:
Possibly dangerous `==` comparison with floating point numbers?

Reply:
In this particular case it's fine, I think, because it's really only this exact unitcell that we are using as the magic marker.

Reply:
Should I revert it back?